src/explore-education-statistics-admin/src/services/releaseDataFileService.ts

```diff
@@ -115,7 +115,7 @@ const releaseDataFileService = {
     data.append('zipFile', request.zipFile);
 
     const file = await client.post<DataFileInfo>(
-      `/release/${releaseId}/zip-data?name=${request.name}`,
+      `/release/${releaseId}/zip-data?name=${encodeURIComponent(request.name)}`,
```

Comment:
I don't think we need to actually do this directly, You can pass a third `options` param to `client.post`, which can contain some `params` (query parameters). Really we should be doing this in lots of other places as well.

Reply:
done